styles.css

```diff
@@ -0,0 +1,156 @@
+/* BACKGROUND */
+body {
+  max-width: 100%;
+  height: auto;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  background: url("./images/background.png") no-repeat fixed center;
+}
+
+/* HEADER */
+
+/* main box */
+.flex-container {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  box-sizing: border-box;
+}
+
+.social-links a {
+  color: #172b4d;
+  text-align: center;
+}
+
+.outer-container {
+  flex-direction: column;
+  margin-top: 2%;
+}
+
+/* main container */
+.main-container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin-top: 9%;
+  width: 100%;
+  padding: 0 17%;
+}
```

Comment:
- [OPTIONAL] Kindly consider removing `17%` padding because try to give it appropriate padding as in the figma design because must stick to the figma design as it is a project requirement.